src/taco-remote/lib/buildManager.ts

```diff
@@ -431,6 +432,17 @@ class BuildManager {
                 } else {
                     self.buildMetrics.failed++;
                 }
+                
+                utils.TelemetryHelper.generate("build",
```

Comment:
The team decided that timing the build isn't a high priority, so I'm not worrying about it.